docs/learn/web-development/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Web Development
+eleventyNavigation:
+  key: Web Development
+  order: 10
+---
+
+Modern browsers are a powerful platform for building websites and applications. We should work with what's available in the browser first before reaching for custom solutions.
```

Comment:
"We should" or "We can"...because they are so powerful? Not sure the underlying goals of the content.

Reply:
Perhaps this should be "Our goal is to"? This is taken from the blog post actually.